src/ruleLoader.ts

```diff
@@ -43,11 +45,18 @@ export function loadRules(ruleConfiguration: {[name: string]: any},
                 const ruleSpecificList = (ruleName in enableDisableRuleMap ? enableDisableRuleMap[ruleName] : []);
                 const disabledIntervals = buildDisabledIntervalsFromSwitches(ruleSpecificList, allList);
                 rules.push(new Rule(ruleName, ruleValue, disabledIntervals));
+            } else {
+                notFoundRules.push(ruleName);
             }
         }
     }
 
-    return rules;
+    if (notFoundRules.length > 0) {
+        const errorMessage = `Could not find the following rules specified in the configuration:\n${notFoundRules.join("\n")}`;
+        throw new Error(errorMessage);
```

Comment:
I like throwing an error because there's no reason you should have a misnamed rule in your config. Also, depending on the context that TSLint is being run in, users may not see the console / log messages. However, you're right, I guess this could cause some troubles for people (although easily fixable ones)